### PR TITLE
[NLU-2872] Fix remaining recognizer tests

### PR DESCRIPTION
**Pull request by samhickey25**

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Definitions/Microsoft.Recognizers.Definitions.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-<?xml version="1.0"?>
```
Comment (ushmau5) [inline .NET/Microsoft.Recognizers.Definitions/Microsoft.Recognizers.Definitions.xml:1]: Don't think you meant to commit anything in the .NET folder?

Comment (samhickey25) [reply .NET/Microsoft.Recognizers.Definitions/Microsoft.Recognizers.Definitions.xml:1]: I don't know how or why these files are even here. There aren't any differences

**Comment by samhickey25** [comment]
> Do the existing tests pass and are there more tests that should be un-skipped?

A lot of Japanese tests are failing atm. Looks to be due to changes from the rebase with MSrecognizers master. Trying to figure out what's going on.
Other than that tests are passing, don't believe there are others to be unskipped

**Comment by ushmau5** [comment]
Since we have made fixes you'd expect that more of the tests can be un-skipped. As for the .NET mystery changes maybe you can revert them so Microsoft don't get confused with them when we open a pull request.

**Review by ushmau5 — APPROVED** [review]

**Review by srandal — APPROVED** [review]

File: Specs/NumberWithUnit/Japanese/CurrencyModel.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ﻿[
   {
     "Input": "今日の日経平均は2222円13銭です。",
+    "NotSupported": "python",
```
Comment (samhickey25) [inline Specs/NumberWithUnit/Japanese/CurrencyModel.json:4]: Looks like these got unskipped in the merges with master.

**Pull request merged**